superset/connectors/sqla/models.py

```diff
@@ -119,6 +119,7 @@
 from superset.superset_typing import (
     AdhocColumn,
     AdhocMetric,
+    Column as ColumnTyping,
```

Comment:
@mayurnewase out of interest why did you rename this? There doesn't seem to be a name clash per your change.

Reply:
@john-bodley there is import of Column from sqlalchemy on line 50.